src/get-printers/get-printers.spec.ts

```diff
@@ -106,3 +123,36 @@ it("fails with an error", () => {
   mockedExecAsync.mockRejectedValue("error");
   return expect(getPrinters()).rejects.toBe("error");
 });
+
+it("returns list of available printers with custom properties", async () => {
+  mockedExecAsync.mockResolvedValue({
+    stdout: mockPrinterListWithPropertiesStdout,
+    stderr: "",
+  });
+
+  const result: Printer[] = await getPrinters();
+
+  expect(result).toStrictEqual([
+    { deviceId: "OneNote", name: "OneNote", paperSizes: [] },
+    {
+      deviceId: "Microsoft-XPS-Document-Writer",
+      name: "Microsoft XPS Document Writer",
+      paperSizes: [],
+    },
+    {
+      deviceId: "Microsoft_Print_to_PDF",
+      name: "Microsoft Print to PDF",
+      paperSizes: [],
+    },
+    {
+      deviceId: "Fax",
```

Comment:
what is the point of checking that all these printers are in the list? i recommend you to replace that list with:
```
{
      deviceId: "canon-printer",
      name: "Canon Printer",
      paperSizes: ["A4", "144mm x 100mm", "2 x 4", "4 x 4"]
}
```

Reply:
I just copied from another test suite, changed to assert only one printer